src/components/Global/MantecaDetailsCard/index.tsx

```
@@ -0,0 +1,23 @@
+    key: React.Key
+}
+
+interface MantecaDetailsCardProps {
+    rows: MantecaCardRow[]
+}
+
+const MantecaDetailsCard: FC<MantecaDetailsCardProps> = ({ rows }) => {
+    return (
+        <Card className="rounded-sm">
+            {rows.map((row) => (
+                <PaymentInfoRow {...row} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)